inttest/etcdmember/etcdmember_test.go

```diff
@@ -71,6 +75,10 @@ func (s *EtcdMemberSuite) TestDeregistration() {
 		}
 	}
 
+	// Let etcd stabilize a bit before we start to mess with it via EtcdMember objects
+	// https://github.com/etcd-io/etcd/blob/5ccbeec769685826424aa147aed452ea4ec52673/server/etcdserver/server.go#L91-L93
+	time.Sleep(5 * time.Second)
```

Comment:
Shouldn't that be handled by retrying things? I mean, if there's no retries this will fail as soon as there's a single hiccup.

Reply:
yes, you're right. I had this in place before I implemented the retry and forgot to remove 👍